explorer/interpreter/interpreter.cpp

```diff
@@ -788,6 +788,10 @@ auto Interpreter::CallFunction(const CallExpression& call,
           alt.alt_name(), alt.choice_name(), arg));
     }
     case Value::Kind::FunctionValue: {
+      if (phase() == Phase::CompileTime) {
+        return ProgramError(call.source_loc())
+               << " function call not allowed here";
```

Comment:
```suggestion
               << " function call not allowed in this context";
```

Reply:
I assume the tests will need to be updated to match this.

Reply:
oh sorry! I forgot!